Telerik.JustMock/Core/PrivateAccessorBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ JustMock Lite
+ Copyright © 2010-2018 Telerik EAD
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Telerik.JustMock.Core
+{
+    public abstract class PrivateAccessorBase
```

Comment:
My comment here will be the same as the one for NonPublicExpecatationBase. Is the functionality in this class enough to call it "PrivateAccessorBase" or is more like enough for a Util class?

Reply:
@lyubomir-rusev, fixed with [595ded1](https://github.com/telerik/JustMockLite/pull/42/commits/595ded10a032d8e33576643110280741ea96a82d)